src/FilterPluginManager.php

```diff
@@ -10,6 +12,29 @@
 use Laminas\ServiceManager\AbstractPluginManager;
 use Laminas\ServiceManager\Exception\InvalidServiceException;
 use Laminas\ServiceManager\Factory\InvokableFactory;
+use Zend\Filter\File\LowerCase;
+use Zend\Filter\File\Rename;
+use Zend\Filter\File\RenameUpload;
+use Zend\Filter\File\UpperCase;
+use zend\filter\word\camelcasetodash;
+use zend\filter\word\camelcasetoseparator;
+use zend\filter\word\camelcasetounderscore;
+use zend\filter\word\dashtocamelcase;
+use zend\filter\word\dashtoseparator;
+use zend\filter\word\dashtounderscore;
+use zend\filter\word\separatortocamelcase;
+use zend\filter\word\separatortodash;
+use zend\filter\word\separatortoseparator;
+use zend\filter\word\underscoretocamelcase;
+use zend\filter\word\underscoretodash;
+use zend\filter\word\underscoretoseparator;
+use zend\filter\word\underscoretostudlycase;
```

Comment:
It looks like PHPCs has broken the imports to the old Zend classes. Any ideas how to solve this?

Reply:
Restored aliases in da43764 and put them in a `@codingStandardsIgnoreStart/End` section, to prevent them from being changed by PHPCs again.